doc/classes/ProjectSettings.xml

```diff
@@ -2345,6 +2345,9 @@
 			[b]Note:[/b] [Control] nodes are snapped to the nearest pixel by default. This is controlled by [member gui/common/snap_controls_to_pixels].
 			[b]Note:[/b] It is not recommended to use this setting together with [member rendering/2d/snap/snap_2d_transforms_to_pixel], as movement may appear even less smooth. Prefer only enabling that setting instead.
 		</member>
+		<member name="rendering/2d/visibility/inherit_visibility_layer" type="bool" setter="" getter="" default="false">
+			If [code]true[/code], [CanvasItem] nodes will inherit the visibility layers of its parent by default.
```

Comment:
Ditto? It's debatable because some project settings only apply at run-time, so "will inherit" would sound more appropriate. But this one also does in the editor (or at least it should), so "inherit" it is.
```suggestion
			If [code]true[/code], [CanvasItem] nodes inherit the visibility layers of their parent nodes by default.
```